Lab.md

```diff
@@ -212,7 +214,7 @@ The next step is to use the Azure Data Factory to create a pipeline to extract t
 
     1. Select the activity in the designer.
     1. In the activity editor pane below the designer, select the **Source** tab, then select **New**.
-    1. Locate the dataset **Office 365**, select it and then select the **Finish** button.
+    1. Locate the dataset **Office 365 (Preview)**, select it and then select the **Finish** button.
```

Comment:
This connector is out of preview with our GA at //build so the preview tag should be removed